FinancialTransactionsApi/V1/Gateways/PostgreDbGateway.cs

```diff
@@ -38,7 +38,31 @@ public async Task<Transaction> GetTransactionByIdAsync(Guid id)
             return await Task.FromResult(response.FirstOrDefault()?.ToDomain()).ConfigureAwait(false);
         }
 
-        public Task<PagedResult<Transaction>> GetPagedTransactionsAsync(TransactionQuery query) => throw new NotImplementedException();
+        public async Task<Paginated<Transaction>> GetPagedTransactionsAsync(TransactionQuery query)
```

Comment:
Does the  `<Paginated<Transaction>>`
implicitly return an `<IEnumerable<Transaction>`?

Reply:
Yes the results on the paginated object returns a "IEnumerable<T> Results"